### ISSUE TITLE
keep tabs instead of spaces

### ISSUE THREAD

**Issue by BertTank** [dotnet/codeformatter#174]
I know this is your default and written down in the documentation: ```We use four spaces of indentation (no tabs)``` but we use tabs by default in my company. However the rest of your rules will perfectly fit ours.

Is it possible to exclude this feature to keep tabs (or even convert spaces to tabs)?

**Comment by genlu** [comment]
@BertTank I believe you can disable the tab formatting by adding `/rule-:FormatDocument` at commandline.

**Comment by BertTank** [comment]
That's right, but I want to have the formatting of that rule - just not with spaces but with tabs.

**Comment by genlu** [comment]
@BertTank Yeah, @michaelcfanning is working on adding solution/project specific options to formatting rules. This space-tab conversion option will definitely be considered.
